src/app/components/shared/datetime/datetime/datetime.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, Input } from "@angular/core";
+import { isInstantiated } from "@helpers/isInstantiated/isInstantiated";
+import { TimezoneInformation } from "@interfaces/apiInterfaces";
+import { DateTime, IANAZone, Zone } from "luxon";
+import { AbstractDatetimeComponent } from "../abstract-datetime.component";
+
+type BawTimezoneUnion = Zone | TimezoneInformation | string;
+
+/**
+ * Displays a datetime (possibly from another timezone) in the users locale and timezone
+ * Commonly used for displaying client side timestamps e.g. Project created at: xyz
+ *
+ * This component does not assume that all dates input are utc, and will localize using ISO8601 offset information if possible
+ */
+@Component({
+  selector: "baw-datetime",
+  templateUrl: "../abstract-datetime.component.html",
+})
+export class DatetimeComponent extends AbstractDatetimeComponent {
+  public constructor() {
+    super();
+  }
+
+  @Input({ required: true }) public value!: DateTime | Date;
```

Comment:
shouldn't this be pushed up to the base class?

Reply:
Yes, fixed.

I originally did this for stricter TypeScript typing (so that each component could define it's value type)

However, by making the AbstractDatetimeComponent a generic abstract class, I was able to define the TS type on the abstract component